pkg/controller/kubedirectorconfig/kubedirectorconfig_controller.go

```diff
@@ -74,8 +76,8 @@ var _ reconcile.Reconciler = &ReconcileKubeDirectorConfig{}
 
 const (
 	// Period between the time when the controller requeues a request and
-	// when it's scheduled again for reconciliation. Zero means don't poll.
-	reconcilePeriod = 0
+	// it's scheduled again for reconciliation. Zero means don't poll.
+	reconcilePeriod = 30 * time.Second
```

Comment:
Without FsMount is there a reason to poll?

Reply:
Nah, not yet. As I mentioned in the PR description this should be harmless, but not useful yet.

I've waffled on whether or not to disable the polling here; I don't want to later merge this change into some other fork that requires the polling and break it.

Maybe if we're both aware of that issue we can make sure it won't happen!